Add io.github.rahatzamancse.ProtonGameSaves

rahatzamancse opened the pull request:
<!-- ⚠️⚠️  Submission pull request MUST be made against the `new-pr` **base branch** ⚠️⚠️  -->

<!-- 💡 Please go to the preview tab to view the markdown below 💡 -->

### Please confirm your submission meets all the criteria

<!-- 💡 Please replace each `[ ]` with `[X]` when the step is complete 💡 -->

<!-- 💡 Please tick and write 'N/A' with a reason if a checklist item below is not applicable 💡 -->

- [X] Please describe the application briefly. 
**Proton Game Saves Manager is a GTK4 application written in Rust that helps you manage save files for Steam games running through Proton compatibility layer. It provides an easy-to-use interface to browse and manage Proton prefix folders, navigate to save file locations for each game, delete unwanted prefixes to free up space, and configure Steam directory location. The application features a modern GTK4/libadwaita UI following GNOME design guidelines and is particularly useful for Steam Deck users and Linux gamers who need to organize their game saves.**

- [X] Please attach a video showcasing the application on Linux using Flatpak. 

https://github.com/user-attachments/assets/b1b31469-a338-4ca7-aaad-5201f6806fe6

- [X] The Flatpak ID follows all the rules listed in the [Application ID requirements][appid].
- [X] I have read and followed all the [Submission requirements][reqs] and the [Submission guide][reqs2] and I agree to them.
- [X] I am the author of the project.

<!-- 💡 Please mention below the GitHub usernames of any additional maintainers needed (if any) 💡 -->

<!-- ⚠️⚠️  Please DO NOT modify anything below this line ⚠️⚠️  -->

[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[reqs2]: https://docs.flathub.org/docs/for-app-authors/submission

rahatzamancse committed:
Add io.github.rahatzamancse.ProtonGameSaves

rahatzamancse committed:
Updated new sha256

bbhtt commented on io.github.rahatzamancse.ProtonGameSaves.json:
```diff
@@ -0,0 +1,53 @@
+{
+    "app-id": "io.github.rahatzamancse.ProtonGameSaves",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "47",
```
48 is latest

bbhtt commented on io.github.rahatzamancse.ProtonGameSaves.json:
```diff
@@ -0,0 +1,53 @@
+        "--socket=fallback-x11",
+        "--share=network",
+        "--own-name=io.github.rahatzamancse.ProtonGameSaves",
+        "--filesystem=~/.local/share/Steam:rw",
+        "--filesystem=~/.steam:rw"
```
why?

bbhtt commented on io.github.rahatzamancse.ProtonGameSaves.json:
```diff
@@ -0,0 +1,53 @@
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/rust-stable/bin",
+        "build-args": [
+            "--share=network"
+        ],
```
not allowed, how do you miss the giant text in the requirements page...

rahatzamancse replied:
Sorry, this is my first submission. Removing it.

rahatzamancse committed:
Update ProtonGameSaves manifest: upgrade runtime and streamline permissions

- Upgrade runtime version from 47 to 48
- Remove unnecessary permissions (network access, D-Bus name ownership)
- Simplify Steam directory access to just ~/.steam
- Fix archive URL format for GitHub release

rahatzamancse committed:
tag checksum updated

hfiguiere commented on io.github.rahatzamancse.ProtonGameSaves.json:
```diff
@@ -41,7 +41,7 @@
                     "type": "archive",
                     "path": "https://github.com/rahatzamancse/ProtonGameSaves/archive/v0.1.0.tar.gz",
-                    "sha256": "81e656723a0284be5e16ed5d734e09fb21ee33eb651d50ff2f256109472e7ced"
+                    "sha256": "c88a60da7659a681a0cbaf3f3c5abc5334ac6888489c4d2c62442cb02bb5a999"
```
retagging is a bad habit. don't do it.

rahatzamancse replied:
Okay, will bump the versions from up next.

rahatzamancse committed:
remove network share

bbhtt commented:
bot, build
